CONTRIBUTING.md

```diff
@@ -3,7 +3,10 @@
 ## Quick guide
 
  * Fork the repo.
- * Checkout the branch you want to make changes on.
+ * Checkout the branch you want to make changes on:
+  * Previous release branch if you are fixing a bug for an existing feature (or the current release branch if the feature you are fixing was introduced in that version)
```

Comment:
Descr for prev branch and then... current branch. And what if bug was made on master?

What about describing it another way:
- master for BC breakers
- current branch for new features
- ...

?